zeppelin-web/bower

```diff
@@ -1,17 +0,0 @@
-#
```

Comment:
@bzz 

I think this cause breaking in term of usage for some people who already use `./grunt test`, `./bower install`

Executing those shell files, however, is not convenient. So I think many users install their own grunt-cli and bower globally so that they can type just `grunt test` and `grunt build` 

IMO, 

- Using locally installed bower and grunt through npm scripts is more natural, common way for front-end users nowdays.
- And if we are moving to webpack from grunt to get reduced build time and other useful things, this is necessary change.

In this case, we need documentation and should notice to our front-end developers